addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -548,6 +556,12 @@ const wSnippetMenu = weSnippetEditor.SnippetsMenu.extend({
     _allowParentsEditors($snippet) {
         return this._super(...arguments) && !$snippet[0].classList.contains("s_popup");
     },
+    /**
+     * @override
+     */
+    _allowInTranslationMode(snippet) {
+        return this._super(...arguments) || !snippet || !!getTextOptionTarget(snippet);
```

Comment:
Seems weird that this function would return "true" if you give a falsy `snippet` value, no?

Reply:
Indeed, as far as I remember, this was a part of an old implementation in which I used `_allowInTranslationMode()` in `_activateSnippet()`, in this case, the check returned `true` to allow normal execution of `_activateSnippet(false)`. I removed it now.